tests/integration/test_replicated_database/test.py

```diff
@@ -16,7 +16,7 @@
 dummy_node = cluster.add_instance('dummy_node', main_configs=['configs/config.xml'], user_configs=['configs/settings.xml'], with_zookeeper=True, stay_alive=True, macros={"shard": 1, "replica": 2})
 competing_node = cluster.add_instance('competing_node', main_configs=['configs/config.xml'], user_configs=['configs/settings.xml'], with_zookeeper=True, macros={"shard": 1, "replica": 3})
 snapshotting_node = cluster.add_instance('snapshotting_node', main_configs=['configs/config.xml'], user_configs=['configs/settings.xml'], with_zookeeper=True, macros={"shard": 2, "replica": 1})
-snapshot_recovering_node = cluster.add_instance('snapshot_recovering_node', main_configs=['configs/config.xml'], user_configs=['configs/settings.xml'], with_zookeeper=True, macros={"shard": 2, "replica": 2})
+snapshot_recovering_node = cluster.add_instance('snapshot_recovering_node', main_configs=['configs/config.xml'], user_configs=['configs/settings.xml'], with_zookeeper=True)
```

Comment:
I didn't get if this test actually test changes. 
Only `snapshot_recovering_node` do not define macros, but I don't see queries that will check that `snapshot_recovering_node` have correct shard and replica name.
Could you clarify it?

Reply:
Replicated table is created on `main_node` [here](https://github.com/ClickHouse/ClickHouse/blob/2f8b8596a243fc99777113884d20a5ac9c789e7f/tests/integration/test_replicated_database/test.py#L269-L271). Then we create new database replica on `snapshot_recovering_node` and check that it successfully created the table [here](https://github.com/ClickHouse/ClickHouse/blob/2f8b8596a243fc99777113884d20a5ac9c789e7f/tests/integration/test_replicated_database/test.py#L284-L292).